tests/functional/test_pubkey_cmd.py

```diff
@@ -45,6 +45,18 @@ def test_get_public_key_show_key_accepted(backend, navigator, test_name):
     check_get_public_key_resp(backend, ZILLIQA_KEY_INDEX, public_key)
 
 
+
+def test_get_public_key_silent(backend, navigator, test_name):
+    client = ZilliqaClient(backend)
+    with client.send_get_public_key_non_confirm(ZILLIQA_KEY_INDEX):
+        pass
+    response = client.get_async_response().data
```

Comment:
can be changed to the following if you change the implementation of `send_get_public_key_non_confirm` :
```suggestion
    response = client.send_get_public_key_non_confirm(ZILLIQA_KEY_INDEX):